extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+| `content.uri` | template URI for the content 3D Models |
+
+Below is a full example of how the tileset JSON file looks in practice:
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "geometricError": 10000,
+  "extensionsUsed": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensionsRequired": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensions": {
+    "3DTILES_implicit_tiling": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "boundingVolume": {
+        "region": [-1.318, 0.697, -1.319, 0.698, 0, 20]
+      },
+      "refine": "REPLACE",
+      "tilingScheme": "QUADTREE",
+      "geometricError": 5000,
+      "subtreeLevels": 7,
+      "maximumLevel": 21,
+      "subtrees": {
+        "uri": "subtrees/${level}/${x}/${y}/subtree.json"
+      },
+      "content": {
+        "mimeType": "model/gltf+json",
+        "uri": "terrain/${level}/${x}/${y}.gltf"
+      }
+    }
+  }
+}
+```
+## Glossary
+
+* **availability** - Data specifying which tiles are available within a single subtree. This helps prevent unnecessary network requests.
+* **available tile** - A tile that exists in the dataset.
+* **boolean array** - An array of boolean values.
+* **bitstream** - A boolean array stored as a sequence of bits rather than bytes.
+* **child subtree** - A subtree reachable from an available tile in the bottommost row of a subtree.
+* **child subtree availability** - Information about what child subtrees are available.
+* **content availability** - Information about which tiles have an associated content file within a single subtree.
+* **explicit tiling** - Describing a tileset by providing information about every tile.
+* **implicit tiling** - Describing a tileset by describing the hierarchical structure in binary rather than in JSON.
+* **octree** - A 3D tiling scheme that divides each cuboid into 8 smaller cuboids.
+* **quadtree** - A 2D tiling scheme that divides each rectangle into 4 smaller rectangles.
+* **root tile** - The topmost tile in a tileset tree.
+* **subtree** - A fixed-depth section of the tileset tree used to break large tilesets into manageable pieces.
+* **subtree file** - A JSON file storing information about a specific subtree.
+* **subtree node** - A node in a subtree, regardless of whether there is an available tile there.
+* **template URI** - A URI pattern containing tile coordinates for directly addressing tiles.
+* **tile** - A tile as described in the Cesium 3D Tiles 1.0 specification.
+* **tile availability** - Information about which tiles exist within a single subtree.
+* **tileset** - A tileset as described in the Cesium 3D Tiles 1.0 specification.
+* **tileset JSON** - A JSON file describing a tileset, as described in the Cesium 3D Tiles 1.0 specification.
+* **tiling scheme** - A well-defined method for subdividing a bounding volume into a hierarchy of tiles.
+* **unavailable tile** - A region of space that does not contain a tile.
+
+## Examples
```

Comment:
I feel like overall we could do better with examples if we had a more visual way of presenting at least one of them. Though as @ErixenCruz and I learned on Friday, putting together labeled examples can be time consuming (since determining morton order is necessary in some diagrams.